atlp-rwanda/trojans-ec-fe
# #184760070 ft-notifications

### **What does this PR do?**
Users Notifications
### **Description of Task to be completed?**
A seller, Admin and buyer when they logged in they can yes the notification like when product added, for seller when  his/her product bought and admin can be able to see all notifications
### **How should this be manually tested?**
to test this feature  you require to login by this user `testBuyer@example.com ` his password is `default` or use other user account  after you can add product to Wishlist to see the notification popup and apply other action like mark as read or delete it. link for my preview
### **Any background context you want to provide?**
N/N
### **What are the relevant pivotal tracker stories?**
https://www.pivotaltracker.com/n/projects/2628461/stories/184760070
### **Screenshots (if appropriate)**
<img width="908" alt="notif 1" src="https://github.com/atlp-rwanda/trojans-ec-fe/assets/112135715/da912e5a-6a0e-49b6-b3c3-d2c2d4a8c429">
<img width="904" alt="notif 3" src="https://github.com/atlp-rwanda/trojans-ec-fe/assets/112135715/9dfb6d83-f923-4f9c-aaac-4781211400d5">
<img width="893" alt="notif 2" src="https://github.com/atlp-rwanda/trojans-ec-fe/assets/112135715/0eb3b40f-51f8-4ec9-af88-8b3d7fb94f99">
